### DISCUSSION TITLE
v4 Migration Guide

### DISCUSSION THREAD

**Discussion by igor-baiborodine** [Lednerb/bilberry-hugo-theme#587] [General]
If you want to migrate your website from `v3` to `v4`, please read the migration [guide](https://github.com/Lednerb/bilberry-hugo-theme/blob/master/v4/v4-migration-guide.md).

**Comment by Sp-cy** [comment]
I've been doing the version migration these days and it's been wonderful. Thank you very much for your hard work! I really love this theme.
